gluon/gluon-mesh-batman-adv-core/check_site.lua

```diff
@@ -7,6 +7,8 @@ for _, config in ipairs({'wifi24', 'wifi5'}) do
    need_string(config .. '.mesh_ssid')
    need_string_match(config .. '.mesh_bssid', '^%x[02468aAcCeE]:%x%x:%x%x:%x%x:%x%x:%x%x$')
    need_number(config .. '.mesh_mcast_rate')
+   need_boolean(config .. '.ssid_disabled')
+   need_boolean(config .. '.mesh_disabled')
```

Comment:
Add `false` as second argument to both `need_boolean` calls so the options aren't required.